AUTHORS.txt

```diff
@@ -0,0 +1,44 @@
+Adrien Devresse 
+Alexandru Săvulescu
+Ali Can Demiralp
+Angelos Plastropoulos
+contre
+Daniel Nachbaur
+Dmitri Bichko
+eudoxos
+Fernando Pereira
+guoxy
+Henry Schreiner
+JaWSnl
+Jia Li
+John W. Peterson
+Jonas Karlsson
+Jorge Blanco Alonso
+Kerim Khemraev
+Marian Heil
+Mario Emmenlauer
+Mathieu Bernard
+Matthias Wolf
+Maximilian Nöthe
+Mightrider
+Mike DePalatis
+Mike Gevaert
+Nico Jahn
+Nicolas Cornu (maintainer)
+Omar Awile
+Pablo Toharia
+Philip Deegan
+Pramod Kumbhar
```

Comment:
Jorge and Pramod have co-authored changes, but they are not listed as contributors in GitHub.

What happens with those contributors? Should we skip them from this list somehow?

Reply:
Add them to the list. We use a '(contributor)' tag if a person has been involved in discussions, but in this case I assume they can be considered authors

Reply:
That is the rule I understood as well. But I wonder if we are missing many other authors...

Reply:
I don't remember contributing anything specific to this project. Co-author might be just because I might have committed some PR? I wouldn't mind just ignoring.